File: lib/cluster_labeling/boostexter_labels/filepaths.py
output_subdir = 'cluster_labeling/cc_boostexter_files/'
boostexter_subdir = 'cluster_labeling/BoosTexter2_1/'

def get_filename_stem(cluster):
    return output_subdir + \
           "%s_%s" % (optemo.product_type.verbose_name,
                      str(cluster.id))

def get_names_filename(cluster):
    filestem = get_filename_stem(cluster)
    filename = filestem + ".names"
    return filename

def get_data_filename(cluster):
    filestem = get_filename_stem(cluster)
    filename = filestem + ".data"
    return filename

def get_strong_hypothesis_filename(cluster):
    filestem = get_filename_stem(cluster)
    filename = filestem + ".shyp"
    return filename    

File: lib/cluster_labeling/boostexter_labels_rb/filepaths.rb
module BtxtrLabels
  Output_subdir = "#{RAILS_ROOT}/lib/cluster_labeling/cc_boostexter_files"
  Boostexter_subdir = "#{RAILS_ROOT}/lib/cluster_labeling/BoosTexter2_1"

  def BtxtrLabels.get_filename_stem(cluster)
    return Output_subdir + "/" + Session.current.product_type + "_" + cluster.id.to_s()
  end

  def BtxtrLabels.get_names_filename(cluster)
    filestem = get_filename_stem(cluster)
    return filestem + ".names"
  end

  def BtxtrLabels.get_data_filename(cluster)
    filestem = get_filename_stem(cluster)
    return filestem + ".data"
  end

  def BtxtrLabels.get_strong_hypothesis_filename(cluster)
    filestem = get_filename_stem(cluster)
    return filestem + ".shyp"
  end
end
